golang/cosmos/x/vstorage/keeper/keeper.go

```diff
@@ -188,6 +189,8 @@ func getEncodedKeysWithPrefixFromIterator(iterator sdk.Iterator, prefix string)
 	return keys
 }
 
+const vstorageMetricKey = "vstorage_size_delta"
```

Comment:
```suggestion
```suggestion
var MetricKeysSizeDelta = []string{"store", "size_delta"}
var MetricLabels = []metrics.Label{
	telemetry.NewLabel("storeKey", k.GetStoreName()),
}
```
```